aemedge/templates/blog-article/blog-article.css

```diff
@@ -247,6 +247,24 @@ a.button.primary:hover
     cursor: none;
 }
 
+.blog-article .default-content-wrapper p,
+.blog-article .default-content-wrapper ul,
+.blog-article .default-content-wrapper li {
+    text-wrap: initial;
+    margin-top: 20px;
+}
+
+.blog-article .default-content-wrapper li {
+    text-wrap: initial;
```

Comment:
this is redundant because it is using text-wrap from line 254

Reply:
added this change to code